keanu-python/tests/test_sampling.py

```diff
@@ -176,6 +176,14 @@ def test_can_specify_a_gaussian_proposal_distribution(net: BayesNet) -> None:
     generate_samples(net=net, sample_from=net.iter_latent_vertices(), sampling_algorithm=algo)
 
 
+def test_can_specify_a_multivariate_gaussian_proposal_distribution(net: BayesNet) -> None:
+    algo = MetropolisHastingsSampler(
```

Comment:
Can you add a test to `MetropolisHastingsTest.java` that uses the proposal distribution? It should be good for the Gaussian sum test cast.